### PR TITLE
Mint-nft-form

### PR DESCRIPTION
**Pull request by sanketdebnath24**
Form for minting nft

### CODE REVIEW

File: src/apis/cryptoApi.js [hunk 1]
```diff
@@ -130,49 +132,42 @@ export const updateBalanceAsync = async ({walletAddress, blockchain}) => {
...
-          name: metadata.name,
-          description: metadata.description,
-          'Background': 'Soft Pink',
-          'Head': 'Head5',
-          'Head-Upper': 'winter cap',
-          'Shirt': 'Shirt1',
-        }),
-      },
-    },
+        data: JSON.stringify(textData)
```
Comment (Vibek-Redsoft) [inline src/apis/cryptoApi.js:154]: Discription is missing

File: src/component/MintNft/MintNftForm/MintNftForm.jsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+import PrimaryButton from '../../common/Buttons/PrimaryButton';
+import ErrorMessage from '../../common/Form/ErrorMessage';
+import TextArea from '../../common/Form/TextArea';
+import SecondaryButton from '../../common/Buttons/SecondaryButton';
+import { mintNft } from '../../../apis/cryptoApi';
+
+const labels = { name: 'name', description: 'description' };
+
+const options = {
+  name: { required: 'This is required' },
```
Comment (Vibek-Redsoft) [inline src/component/MintNft/MintNftForm/MintNftForm.jsx]: The validation messages different even though they mean the same, keep it simple like "Please provide ..."

File: src/component/MintNft/MintNftForm/MintNftForm.jsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+          walletAddress,
+          blockchain
+        };
+      };
+      (async function () {
+        const walletObj = await getWallet();
+        setWallet(walletObj);
+      })();
+    } catch (error) {
+      console.log(error);
```
Comment (Vibek-Redsoft) [inline src/component/MintNft/MintNftForm/MintNftForm.jsx:62]: Handle this error with the toast

File: src/component/MintNft/MintNftForm/MintNftForm.jsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    });
+
+  async function convert(file) {
+    const dataURI = await toBase64(file);
+    return dataURI;
+  }
+
+  const onSubmit = async (data) => {
+    try {
+      console.log(data);
```
Comment (Vibek-Redsoft) [inline src/component/MintNft/MintNftForm/MintNftForm.jsx]: Remove all logs

**Review by Vibek-Redsoft — APPROVED** [review]

File: src/apis/cryptoApi.js (hunk 1 above)
Comment (sanketdebnath24) [reply src/apis/cryptoApi.js:154]: Text data Structure  {
    name,
    description,
    dynamicKeyProvidedByUser: valueProvidedByUser
}
Check Line86 MintNFTForm.

**Pull request merged**